scripts/checks.sh

```diff
@@ -409,6 +408,35 @@ function check_disk_expansion()
 	fi
 }
 
+function check_image_corruption()
+{
+	local images
+	local corrupted=()
+	local -i corrupted_count=0
+	# TODO: this command is filtering out any images with a size <1Kb (for delta-based images)
+	images=$(${TIMEOUT_CMD} ${ENG} image ls --format "{{.ID}} {{.Size}}" | awk '/![1-9]*B/{next;} {print $1}' | sort -u)
```

Comment:
My awk-fu is bad. Can I get some pointers?

Reply:
Sure! We're matching `[0-9]+B$` (any image between 0B and 1kB), but the `{next;}` instead skips that match. We then print the image ID.